js/src/scraper/scraper.link.js

```diff
@@ -0,0 +1,28 @@
+var scrapers = require( "./../scraper-store.js" );
+
+var Scraper = function() {};
+
+Scraper.prototype.scrape = function(fields){
+
+    var that = this;
```

Comment:
I looks like this variable isn't used in this method. You might consider removing it.